src/base/ui_object.js

```diff
@@ -122,10 +122,10 @@ export default class UIObject extends BaseObject {
 
   /**
    * removes the ui component from DOM
-   * @method remove
+   * @method destroy
    * @return {UIObject} itself
    */
-  remove() {
+  destroy() {
     this.$el.remove()
     this.stopListening()
```

Comment:
What do you think about moving ```this.stopListening()``` to the *BaseObject*  and calls here the method ```super.destroy()```? With this change, we can also remove the method ```destroy``` from the others components that extends from ***BaseObject***.

Reply:
@afonsocdaniel as we agreeed, let's do this as a separate change.